nixos/tests/vmware-guest.nix

```diff
@@ -0,0 +1,82 @@
+import ./make-test-python.nix (
+  { lib, pkgs, ... }:
+  {
+    name = "vmware-guest";
+    meta.maintainers = with lib.maintainers; [ kjeremy ];
+
+    nodes = {
+      headless =
+        {
+          config,
+          options,
+          pkgs,
+          ...
+        }:
+        {
+          virtualisation.vmware.guest = {
+            enable = true;
+            headless = true;
+          };
+
+          environment.variables = {
+            EXPECTED_VMWARE_TOOLS_PATH = pkgs.open-vm-tools-headless;
+            VMWARE_TOOLS_PATH = config.virtualisation.vmware.guest.package;
+          };
+        };
+
+      full =
+        {
+          config,
+          options,
+          pkgs,
+          ...
+        }:
+        {
+          virtualisation.vmware.guest = {
+            enable = true;
+            headless = false;
+          };
+
+          environment.variables = {
+            EXPECTED_VMWARE_TOOLS_PATH = pkgs.open-vm-tools;
+            VMWARE_TOOLS_PATH = config.virtualisation.vmware.guest.package;
+          };
+        };
+
+      custom =
+        {
+          config,
+          options,
+          pkgs,
+          ...
+        }:
+        {
+          virtualisation.vmware.guest = {
+            enable = true;
+            headless = true;
+
+            # Override the package with non-default
```

Comment:
but that's the default

Reply:
Not in the case where `headless` is true.